.github/ISSUE_TEMPLATE/project-pitch.yml

```diff
@@ -5,8 +5,35 @@ labels: ['pitch']
 body:
   - type: markdown
     attributes:
-      value: |
-        Thank you for submitting a project to Brainhack Western! Before you
+      value: >
+        Thank you for submitting a project to Brainhack Western! As a project lead,
+        you'll be working with your team of hackers on a neuroscience project in
+        competition for some exciting prizes. Your project can be anything you want, with
```

Comment:
I would drop the mention of prizes -- I think we generally keep that vaguely under wraps until projects have been pitched